lib/quicly.c

```diff
@@ -116,6 +116,10 @@ struct st_quicly_pn_space_t {
      *
      */
     int64_t send_ack_at;
+    /**
+     * packet count before ack is sent
+     */
+    uint8_t unacked_count;
```

Comment:
How about using `uint32_t` to avoid overflow?

I understand that overflow of `unacked_count` does not cause an issue in the particular way we use it, but it's confusing.

Note that due to other members of the struct, the size of the struct will be aligned to 8-bytes or larger, so using `uint32_t` (or `uint64_t`) will not lead to using more memory on a 64-bit platform.

Reply:
Makes sense, done.